### DISCUSSION TITLE
AssertionError: PROCESSES in Unified Planning Colab Code

### DISCUSSION THREAD

**Discussion by juancarlosgss** [aiplan4eu/unified-planning#630] [Q&A] [answered]
Hi everyone,

I’ve been working on a code in Colab using Unified Planning, and it was working perfectly until recently. Suddenly, it started throwing the following error:

python

AssertionError Traceback (most recent call last)

<ipython-input-3-b46043a5d57b> in <cell line: 1>()
----> 1 Counter = UserType('Counter')
 
      2 

      3 value = Fluent('value', IntType(), m=Counter)

8 frames

/usr/local/lib/python3.10/dist-packages/unified_planning/model/problem_kind.py in _set(self, feature, possible_features)
    157     def __new__(cls, name, bases, dct):
    158         def _set(self, feature, possible_features):
--> 159             assert feature in possible_features, str(feature)
    160             added_feature_version, _ = FEATURES_VERSIONS.get(feature, (1, None))

AssertionError: PROCESSES

I also tried running an example from the Unified Planning Colab page (which used to work), and it's now generating the same error. Does anyone have insight into what might be causing this? Here's the link to the example: https://unified-planning.readthedocs.io/en/latest/notebooks/01-numeric-planning.html.

Thanks in advance!

**Comment by mikand** [comment]
@juancarlosgss Is it possible that you are not using the main branch, but switched to #629 ?

@hstairs Another possibility is that you updated up-enhsp upstream on pip?

**Comment by mikand** [comment, marked as answer]
I checked and the issue is in up-enhsp.

@juancarlosgss if you force the previous version of up-enhsp, you should be able to workaround the issue:
```
pip install up-enhsp==0.0.25
```

**Reply by juancarlosgss** [reply]
thank you very much!
